package.json

```diff
@@ -68,7 +68,6 @@
     "@types/chai": "^4.2.15",
     "@types/express": "^4.17.12",
     "@types/is-url": "^1.2.29",
-    "@types/mailgun-js": "^0.22.11",
```

Comment:
Is this related with npm CI (ts) error?
![image](https://user-images.githubusercontent.com/19143644/128669109-0fdbfa47-a889-4ade-a905-a419dbe88cc1.png)

Reply:
fixed. missing dependencies for mailgun.js